packages/rlc-common/src/metadata/buildPackageFile.ts

```diff
@@ -57,6 +58,14 @@ export function buildPackageFile(
     packageInfo = buildAzureStandalonePackage(extendedConfig);
   }
 
+  if (
```

Comment:
could you move this logic to buildAzureMonorepoPackage function?

Reply:
this change should be an incremental update for typespec-ts.